xblock/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+Useful classes and functionality for building and testing XBlocks
+"""
+
+__version__ = '3.4.1'
```

Comment:
We don't need an independent version for the utils anymore.  I think we can drop this line.